src/index.ts

```diff
@@ -39,6 +40,15 @@ window.addEventListener('load', () => {
   const layoutParam = urlSearchParams.get('layout');
   const layout = layoutParam ? layoutParam : 'vertical-scroll';
 
+  const firebaseSyncParam = urlSearchParams.get('firebase-sync');
+  let firebaseSync = firebaseSyncParam ? firebaseSyncParam === 'true' : false;
```

Comment:
```javascript
const firebaseSync = firebaseSyncParam === 'true' ? firebaseInit() : false;
```

It's fine to compare undefined to the "true" string, triple equality operator will guard against unintentional type conversions.

Reply:
That would mean that if the user sets 'firebase-sync=false', it would still end up enabled

Reply:
done